tests/test_custom_scalars.py

```diff
@@ -24,23 +24,23 @@ def serialize(date):
 
 
 def parse_literal(ast):
-    if not isinstance(ast, StringValue):
-        return None
+    if not isinstance(ast, StringValueNode):
+        raise ValueError()
 
     formatted_date = ast.value
     try:
         parsed_datetime = datetime.strptime(formatted_date, "%Y-%m-%d")
         return parsed_datetime.date()
     except ValueError:
-        pass
+        raise ValueError()
```

Comment:
Why catch and raise the same exception?

Reply:
I've updated the docs to provide explanation:

> If we didn't swap the original exception, it's message would be included by the GraphQL at the end of generated message, possibly **leaking implementation details** to the client.

Reply:
I've updated our tests and docs.

Reply:
Couldn't we add a Python comment as well?